ServerlessLibraryAPI/Models/LibraryItem.cs

```diff
@@ -3,11 +3,35 @@
 
 namespace ServerlessLibrary.Models
 {
+    public class LibraryItemWithStats : LibraryItem
+    {
+        internal static LibraryItemWithStats Load(LibraryItem libraryItem)
+        {
```

Comment:
where is this used? this code is similar to ConvertTo<T>()

Reply:
Missed this. thanks!